### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Added Wishlist page in trendy-t E-Commerce.
Screenshot (286)

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
good to go

File: src/Pages/Wishlist-page/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.wishlist-page {
+  font-family: "Montserrat", sans-serif;
+  display: grid;
+  grid-template-rows: 2.5rem minmax(calc(100vh - 7.5rem), 1fr) 5rem;
+  grid-template-areas:
+    "header"
+    "main"
+    "footer";
```
Comment (JantuDeb) [inline src/Pages/Wishlist-page/wishlist.css:8]: I like the way you are using grid template area.

File: src/Pages/Wishlist-page/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+::-webkit-scrollbar {
+  width: 5px;
+  height: 5px;
+}
+::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 10px rgb(255, 255, 255);
+}
+
+::-webkit-scrollbar-thumb {
+  background-color: var(--primary-color);
```
Comment (JantuDeb) [inline src/Pages/Wishlist-page/wishlist.css]: As you are changing browser default style I suggest you to add this style for other browser also at least  for Mozilla

**Review by JantuDeb — APPROVED** [review]

File: src/Pages/Wishlist-page/wishlist.css (hunk 1 above)
Comment (SaurabhChawda) [reply src/Pages/Wishlist-page/wishlist.css:8]: Thankyou

File: src/Pages/Wishlist-page/wishlist.css (hunk 2 above)
Comment (SaurabhChawda) [reply src/Pages/Wishlist-page/wishlist.css]: > 

Added scrollbar for firefox

**Pull request merged**